src/tests/test_protocol.py

```diff
@@ -4,17 +4,14 @@
 
 import unittest
 
-from .common import skip_python3
```

Comment:
try re-adding these lines for debugging if you still can't find out why it's failing in p3